samcli/commands/_utils/options.py

```diff
@@ -788,3 +789,18 @@ def _space_separated_list_func_type(value):
 
 
 _space_separated_list_func_type.__name__ = "LIST"
+
+
+def generate_next_command_recommendation(command_tuples: List[tuple]):
```

Comment:
A tighter type would be (needs `typing.Tuple`):

```suggestion
def generate_next_command_recommendation(command_tuples: List[Tuple[str, str]]) -> str:
```

Otherwise it'll accept accept any tuple and the command will fails.

Reply:
Implemented!